spec/outputs/logentries_spec.rb

```diff
@@ -0,0 +1,2 @@
+require "logstash/devutils/rspec/spec_helper"
```

Comment:
I see no test here, so I would recommend adding test before we merge this PR. If you @siniar1990 don't know how to proceed here with testing we can work with you figuring out testing ideas.